src/com/lftechnology/java/training/dipak/employeemanagement/dao/DbConnection.java

```diff
@@ -0,0 +1,49 @@
+
+package com.lftechnology.java.training.dipak.employeemanagement.dao;
+
+import java.sql.*;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * <p>This class has a static method for establishing database connection.</p>
+ * 
+ * @author Dipak Thapa<dipakthapa@lftechnology.com>
+ */
+public class DbConnection {
+	
+	private static final Logger LOGGER =
+		Logger.getLogger(DbConnection.class.getName());
+	
+	
+	private DbConnection() {
+	}
+	/**
+	 * <p>This method is used to set database connection.</p>
+	 * 
+	 * @return connection
+	 */
+	public static Connection setConnection() {
+
+		final String USER_NAME = "root";
+		final String PASSWORD = "liferay";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00117)